packages/amplication-git-service/src/Dto/entities/GithubFile.ts

```diff
@@ -12,7 +9,7 @@ export class GithubFile {
   path: string | null;
 
   @Field(() => String)
-  content: string;
+  content!: string;
```

Comment:
I was asking myself the same, and forgot to include it in the review. Thanks!

Reply:
Yes, we know that we get the content of the file as a string if the file exists if not, it will throw.
Its let us write `amplicationignore.content.split('\n') || [];` instead of `amplicationignore?.content?.split('\n') || [];`

Reply:
Not all files have a string value though. Some files don't follow the `utf-8` format. For example binary executable and images that we might want to export which will be buffers.

I suggest `amplicationignore?.content?.split('\n') || [];` for future proofness. At the end of the day, this class is called `GithubFile`, not `GithubTextFile`.